index.html

```diff
@@ -1,15 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Book Collection</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <div id="book-app">
-    <div id="book-list">
+  <nav class="desktop-nav">
+    <h1>Awesome Books</h1>
+    <ul>
+      <li><a href="#books-list">Books List</a></li>
+      <li><a href="#add-book-form">Add New</a></li>
+      <li><a href="#contact-info">Contact Info</a></li>
+    </ul>
+  </nav>
+  
+  <nav class="mobile-nav">
+    <h1>Awesome Books</h1>
+    <svg viewBox="0 0 100 80" width="40" height="40" class="svg-menu" id="svg-menu">
+      <rect y="38" width="100" height="8" rx="8" fill="#008080"></rect>
+      <rect y="58" width="100" height="8" rx="8" fill="#008080"></rect>
+      <rect y="78" width="100" height="8" rx="8" fill="#008080"></rect>
+    </svg>
+    <nav id="mobile-nav-menu" class="mobile-menu-inactive">
+      <div class="disabled-button"><a href="#"><img src="./images/hamburger disable.png"></a></div>
+      <ul class="mobile-menu-list">
+        <li><a href="#books-list" class="mobile-link">Books List</a></li>
+        <li><a href="#add-book-form" class="mobile-link">Add New</a></li>
+        <li><a href="#contact-info" class="mobile-link">Contact Info</a></li>
+      </ul>
+    </nav>
+  </nav>
+  
+  <main>
+    <section id="books-list" class="content-section">
       <h2 class="title-text">All Awesome Books</h2>
       <table id="book-table">
         <thead>
```

Comment:
- [ ] Good job, @Team However, I think your design could be greatly improved if the contents (list of books) are centered in the table. It will really make the page more aesthetic :+1: 
Check this:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/235933091-b12ac401-7e6e-4eb1-b42f-8ad7448681c9.png)|![image](https://user-images.githubusercontent.com/99715640/235933363-3757e0af-2f27-45ee-9606-af1c27f563c5.png)|

- **_Kindly align the contents to the center of the page so that it fits well with the already centralized title (All Awesome Books)_**
- _Just a reminder on implementing the time & date feature, in case you haven't... :thought_balloon:_